synapse/rest/client/v2_alpha/capabilities.py

```diff
@@ -61,6 +61,22 @@ async def on_GET(self, request: SynapseRequest) -> Tuple[int, JsonDict]:
                 "org.matrix.msc3244.room_capabilities"
             ] = MSC3244_CAPABILITIES
 
+        # Add this to response only if `enable_set_displayname` is set to `false` (not default)
+        # because it is experimental
+        # This needs an update after release of MSC3283. Then it will be always available.
+        if not self.config.enable_set_displayname:
+            response["capabilities"]["org.matrix.msc3283.set_displayname"] = {
+                "enabled": self.config.enable_set_displayname
+            }
+        if not self.config.enable_set_avatar_url:
+            response["capabilities"]["org.matrix.msc3283.set_avatar_url"] = {
+                "enabled": self.config.enable_set_avatar_url
+            }
+        if not self.config.enable_3pid_changes:
+            response["capabilities"]["org.matrix.msc3283.3pid_changes"] = {
+                "enabled": self.config.enable_3pid_changes
+            }
+
```

Comment:
As I wrote [in the MSC](https://github.com/matrix-org/matrix-doc/pull/3283#discussion_r678371872), I'm surprised that these are omitted if the setting is true.

Reply:
This was because the missing `experimental` config option. The idea was to  hide experimental keys by default. Maybe I had misunderstood anoa in a private talk.

Reply:
I have changed it.